src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
```

Comment:
It is tricky to get right, I am aware that there are some messages at the top of the dlgTriggerEditor.cpp file that do or should get painted in the "systemMessageArea" that `dlgTriggerEditor::mpSystemMessageArea` is the handle for and which `dlgTriggerEditor::showInfo( QString )`, `dlgTriggerEditor::showWarning( QString )` and `dlgTriggerEditor::showError( QString )` use.  Those messages are currently of low quality and generally include a list of numbered steps which would benefit from revision to include laying out as a numbered list (using HTML tags?) but which will use more vertical space {which is OK as they appear only when there is nothing else to show on the right hand side!}

As I see it the bottom line is ideally I need to:
* find a way to align the systemMessageArea to the top of the dialogue
* to have a pleasingly proportioned minimum size (more like the 2.1 sample immediately above here).
* to allow it to grow reasonably should the system message text get larger.
* to convert embedded line feeds into \<p>\</p> so that they get reproduced in multi-line error message from the Lua sub-system {`require`ing a lua library that is not found produces the biggest error messages I believe} :open_mouth:

Note I already found that it was necessary to parse error messages for `<` and `>` as otherwise the HTML parser hides things like `<eof>` that the lua system sometimes throws up in its error messages...!